docs/disaster-recovery/full-recovery.md

```
@@ -0,0 +1,740 @@
+sudo docker swarm init --force-new-cluster --advertise-addr ${RECOVERY_HOST_IP}
+```
+
+Verify:
+
+```bash
+sudo docker node ls
+```
+
+Example output:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)